mocks_schema/sqitch.plan

```diff
@@ -1,5 +1,6 @@
 %syntax-version=1.0.0
 %project=mocks
+%uri=mocks
```

Comment:
is sqitch complaining about having multiple projects without a URI? If so this one should probably be `https://github.com/bcgov/cas-ciip-portal/tree/develop/mocks_schema`